contribution/contribution.js

```diff
@@ -10,20 +28,18 @@ function contribute(contributions, rand) {
 
     const G1 = bls.bls12_381.CURVE.G1;
     const G2 = bls.bls12_381.CURVE.G2;
-    const Fr = bls.bls12_381.CURVE.Fr;
 
     const util = bls.bls12_381.utils;
 
     for(var i = 0; i < contributions.length; i++) {
         const g1Powers = contributions[i].powersOfTau.G1Powers;
         const g2Powers = contributions[i].powersOfTau.G2Powers;
-
-        var xi = 1n;
+        var xi = BigInt(1);
```

Comment:
afaik, this is same to previous one. why we should change to this?

Reply:
It will help to distinguish it clearly.